src/api.cpp

```diff
@@ -2689,8 +2689,11 @@ cl_int CLVK_API_CALL clGetKernelWorkGroupInfo(
         copy_ptr = &val_ulong;
         ret_size = sizeof(val_ulong);
         break;
+    case CL_KERNEL_COMPILE_WORK_GROUP_SIZE:
+        copy_ptr = kernel->required_work_group_size().data();
+        ret_size = sizeof(size_t[3]);
```

Comment:
In most (all?) other places, size calculations use `size()` on the container * sizeof(type). I'm not saying one is better than the other, just pointing out the very minor style divergence.